scripts/provisioning/s3_setup

```diff
@@ -195,27 +216,21 @@ configure_openldap()
         /opt/seagate/cortx/s3/install/ldap/replication/setupReplicationScript.sh -h hosts_list_file.txt -p "$rootdnpasswd"
         if [ $? -ne 0 ]
         then
-            echo "ERROR: failed to configure ldap replication, exiting."
+            echo "ERROR: Failed to configure ldap replication, exiting."
+            echo "ERROR: You may want to enable openldap logs, and check what exactly failed."
+            echo "ERROR: Run 's3_setup cleanup', followed by 's3_setup config' again, after resolving the issue."
             exit 1
         fi
         rm -f hosts_list_file.txt
     else
-        echo "INFO: Not setting ldap-replication as 'cluster>node_count' is less than 2."
+        echo "INFO: Not setting ldap-replication as the cluster node_count is less than 2."
     fi
-
-    if [[ -f /opt/seagate/cortx/s3/install/ldap/rsyslog.d/slapdlog.conf ]]
-    then
-        # Enable slapd logging in rsyslog config
-        mkdir -p /etc/rsyslog.d
-        cp /opt/seagate/cortx/s3/install/ldap/rsyslog.d/slapdlog.conf /etc/rsyslog.d/slapdlog.conf
-    fi
-    systemctl restart s3authserver
-    systemctl restart rsyslog
+    echo "INFO: Successfully configured openldap on the node."
 }
 
 add_s3backgrounddelete_rabbitmq_user()
 {
-    users=`rabbitmqctl list_users`
+    users=$(rabbitmqctl list_users)
```

Comment:
what if users is empty

Reply:
Incase "cortx-s3" user does not exist, or "users" is empty, then we create "cortx-s3" user (Most common scenario)

But if the "users" is not empty and "cortx-s3" user already exists, we first delete "cortx-s3" user, and then go ahead and create "cortx-s3" user again.